File: Chapter9/i.cpp
#include <iostream>

using namespace std;

int getFib(int n) {
    if (n <= 1)
        return n;
    return getFib(n - 1) + getFib(n - 2);
}

int main() {
    int n;
    cin >> n;

    cout << getFib(n) << '\n';
    return 0;
}

File: Chapter9/i.pas
var
    n: Integer;


function get_fib(n: Integer): Integer;
    begin
        if (n <= 1) then
            get_fib := n
        else
            get_fib := get_fib(n - 1) + get_fib(n - 2);
    end;

Begin
    readln(n);
    writeln(get_fib(n));
End.
